src/csharp/Microsoft.Spark/Versions.cs

```diff
@@ -16,7 +16,7 @@ internal static class Versions
 
         // The following is used to check the compatibility of UDFs between
         // the driver and worker side. This needs to be updated only when there
-        // is a breaking change on the UDF contract.
-        internal const string CurrentVersion = "0.9.0";
+        // is a breaking change (major version change) on the UDF contract.
+        internal const string CurrentVersion = "1.0.0";
```

Comment:
Is this version going to be tightly coupled with the Microsoft.Spark assembly version?  If so then can we just get the version of Microsoft.Spark loaded in the the current `AppDomain` ?

Reply:
Yes, good point. Now that we are going with semantic versioning, we don't need this. I will rework on this after #715 is in.